server/index.js

```diff
@@ -2,10 +2,21 @@ const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
 const port = process.env.PORT || 3007;
+const db = require('../database/index.js');
 
+app.use(express.static('./public'))
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-
+//write the router functions to call the db/index functions
+app.get('/:id', (req, res) => {
+  db.getExpReviews(req.params.id, (err, reviews) => {
+    if (err) {
+      res.status(400).send(err);
+    } else {
+      res.status(200).send(reviews);
```

Comment:
Do the reviews (which from your callback, it's the response that you get back) need to be json parsed or does bodyParser handle it?

Reply:
In postman, I get a JSON object returned (see below):
<img width="899" alt="Screen Shot 2020-01-24 at 11 54 36 AM" src="https://user-images.githubusercontent.com/39776765/73099400-4fa31a00-3ea0-11ea-91b2-9a0c19e46c93.png">